### PR TITLE
Live debugging doc

### LINKED ISSUE #833: Live debugging doc

### PR DESCRIPTION
**Pull request by wildum**
#### PR Description

This PR adds the live debugging doc to the live debugging feature branch.

#### Which issue(s) this PR fixes

Fixes #833 

#### Notes to the Reviewer

Just like the two previous PRs for the back-end and the front-end, this PR is merged to a feature branch. The feature branch will then be merged to main once the full feature is ready.

#### PR Checklist

- [x] Documentation added

### CODE REVIEW

**Review by clayton-cornell — COMMENTED** [review]
I noticed that the button text (in the associated PRs) is `Live debugging`  and the page title is `Live Debugging`.  I suggested `Live debugging` here in the doc... we should align everywhere.

File: docs/sources/tasks/debug.md [hunk 1]
```diff
@@ -70,12 +70,30 @@ The clustering page shows the following information for each cluster node:
 * The node's current state (Viewer/Participant/Terminating).
 * The local node that serves the UI.
 
+### Live Debugging page
+
+<!-- TODO: add screenshot once the branch is ready to go to main -->
+
+The live debugging provides a real-time stream of debugging data from a component. You can access this page from the corresponding [Component detail page](#component-detail-page).
```
[the remark this answers is not in the document]
Comment (wildum) [reply docs/sources/tasks/debug.md]: I added a line about it, please reformulate it if needed

File: docs/sources/tasks/debug.md [hunk 2]
```diff
@@ -70,12 +70,30 @@ The clustering page shows the following information for each cluster node:
...
+
+Please note that not all components support this feature yet. We are working on gradually extending this capability to more components.
+
 ## Debugging using the UI
 
 To debug using the UI:
 
 * Ensure that no component is reported as unhealthy.
 * Ensure that the arguments and exports for misbehaving components appear correct.
+* Ensure that the live debugging data meets your expectations.
```
[the remark this answers is not in the document]
Comment (wildum) [reply docs/sources/tasks/debug.md]: debugging data sounds good

File: docs/sources/tasks/debug.md [hunk 3]
```diff
@@ -57,6 +58,8 @@ The component detail page shows the following information for each component:
 * The current exports for the component.
 * The current debug info for the component (if the component has debug info).
 
+From there you can also jump to the documentation page of the component or to its corresponding [Live Debugging page](#live-debugging-page).
```
Comment (clayton-cornell) [inline docs/sources/tasks/debug.md:61]:
```suggestion
From there you can also go to the component documentation or to its corresponding [Live Debugging page](#live-debugging-page).
```

**Review by clayton-cornell — APPROVED** [review]
One minor suggestion.

**Pull request merged**